tests/phpunit/tests/media.php

```diff
@@ -5992,6 +5998,277 @@ static function ( $loading_attrs ) {
 		);
 	}
 
+	/**
+	 * Test generated markup for an image with lazy loading gets auto-sizes.
+	 *
+	 * @ticket 61847
+	 */
+	public function test_image_with_lazy_loading_has_auto_sizes() {
+		$this->assertStringContainsString(
+			'sizes="auto, ',
+			wp_get_attachment_image( self::$large_id, 'large', false, array( 'loading' => 'lazy' ) ),
+			'Failed asserting that the sizes attribute for a lazy-loaded image includes "auto".'
+		);
+	}
+
+	/**
+	 * Test generated markup for an image without lazy loading does not get auto-sizes.
+	 *
+	 * @ticket 61847
+	 */
+	public function test_image_without_lazy_loading_does_not_have_auto_sizes() {
+		$this->assertStringContainsString(
+			'sizes="(max-width: 1024px) 100vw, 1024px"',
+			wp_get_attachment_image( self::$large_id, 'large', false, array( 'loading' => '' ) ),
```

Comment:
We should use `false` instead of `''` to omit the `loading` attribute.
```suggestion
			wp_get_attachment_image( self::$large_id, 'large', false, array( 'loading' => false ) ),
```